exercises/react/src/example.c

```diff
@@ -0,0 +1,210 @@
+#include <stdlib.h>
+#include "react.h"
+
+enum cell_kind {
+   kind_input,
+   kind_compute1,
+   kind_compute2,
+};
+
+struct child {
+   struct cell *cell;
+   struct child *next;
+};
+
+struct cb {
+   callback_id id;
+   callback f;
+   void *obj;
+   struct cb *next;
+};
+
+struct reactor {
+   struct child *first;
+   struct child *last;
```

Comment:
Whoops, I didn't look closely enough and only took note of the fact that `->last` was on the left side of the assignment.

I think you could add at the head of the list, but then you'd have to traverse in reverse order when you do an update or dependencies won't be updated before dependents. My 2 cents: leave it as-is.